README.md

```diff
@@ -68,9 +68,9 @@ pinyin('汉语拼音', { pattern: 'final' }); // 'àn ǔ īn īn'
 | 对比项              | pinyin           | pinyin-pro | @napi-rs/pinyin |
 | ------------------ | ---------------- | ---------- | --------------- |
 | 准确率              | 97.844%          | 99.744%    | 97.433%         |
-| 5k 字转换所需时间    | 4.879ms          | 6.648ms    | 12.386ms        |
-| 1w 字转换所需时间    | 11.240ms         | 18.315ms   | 27.468ms        |
-| 100w 字转换所需时间  | 392.283ms        | 624.481ms  | 3.782s          |
+| 5k 字转换所需时间    | 489.252ms        | 5.909ms    | 115.723ms       |
+| 1w 字转换所需时间    | 511.573ms        | 15.260ms   | 115.887ms       |
+| 100w 字转换所需时间  | 内存溢出转换失败    | 595.131ms  | 570.994s        |
```

Comment:
Updated the benchmark results with the option `segment: true`.
`pinyin` failed to complete it. Interestingly, `@napi-rs/pinyin` runs faster with `segment: true` than its default.